doorman/templates/email/alert.body.txt

```diff
@@ -1,9 +1,9 @@
 A doorman alert was triggered: {{ rule.name }}
 
 Timestamp: {{ timestamp }}
-   Action: {{ match.action }}
+   Action: {{ match.result['action'] }}
```

Comment:
Would prefer to leave dict style notation out, and instead stick with attribute access via `{{ match.result.action }}`.